jupyterhub/jupyterhub/overlays/build/jupyterhub-img-imagestream.yaml

```diff
@@ -14,4 +14,4 @@ spec:
       name: quay.io/odh-jupyterhub/jupyterhub-img:v0.2.8
```

Comment:
Since we are building the image locally, you can remove `.spec.tags` in the `build` overlay since it will be populated by output the `jupyterhub-img` image build

Reply:
Ok perfect I didn't think about that, although now that I look into `jupyterhub-imagestream.yaml` it makes totally sense. Thank you very much!